flexmeasures/cli/data_add.py

```diff
@@ -941,6 +941,20 @@ def create_schedule(ctx):
     required=False,
     help="Optimize production against this sensor. Defaults to the consumption price sensor. The sensor typically records an electricity price (e.g. in EUR/kWh), but this field can also be used to optimize against some emission intensity factor (e.g. in kg CO₂ eq./kWh). Follow up with the sensor's ID.",
 )
+@click.option(
+    "--consumption-price-sensor-per-device",
+    "consumption_price_sensor_per_device",
+    type=dict,
```

Comment:
I'm interested to know how you pass a dict to a CLI option. I sincerely hope it's possible, but I haven't seen it before.

Maybe [it is possible somehow](https://stackoverflow.com/a/47730333), but it doesn't seem trivial.

Reply:
Good catch. I guess passing multiple pairs of sensor IDs and then building the dict inside the CLI function would be a better design.

Pairs could be two IDs separated by a colon, e.g. "12:4".